staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -27,6 +27,7 @@ import (
 type backoffEntry struct {
 	backoff    time.Duration
 	lastUpdate time.Time
+	retry      int32
```

Comment:
Can you pull this entire backoff.go change into a separate pull and get a review from the usual client suspects (chao, daniel, myself) with a good description of what its buying you. My initial impression is that is exposing internals that I wouldn't expect to see leaked.

Reply:
@deads2k thanks for the review, I created the separated pull request: https://github.com/kubernetes/kubernetes/pull/51626